src/main/java/Nanami.java

```diff
@@ -0,0 +1,289 @@
+package src.main.java;
+
+import src.main.java.task.Deadline;
+import src.main.java.task.Event;
+import src.main.java.task.Task;
+import src.main.java.task.ToDo;
+import src.main.java.taskExceptions.DeadlineMismatchedParameterException;
+import src.main.java.taskExceptions.EmptyTaskException;
+import src.main.java.taskExceptions.EventMismatchedParameterException;
+import src.main.java.taskExceptions.ToDoMismatchedParameterException;
+
+import java.io.*;
+import java.util.Arrays;
+
+public class Nanami {
+    private static Task[] tasklist = new Task[100];
+    private static int taskcount;
+    private int taskCount = 0;
```

Comment:
is there a difference between these variable? it seems like taskCount variable is not used in your code!